test/multiverse/suites/grpc/helper_test.rb

```diff
@@ -0,0 +1,61 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'newrelic_rpm'
+
+class GrpcHelperTest < Minitest::Test
+  include MultiverseHelpers
+
+  class HelpedClass
+    include NewRelic::Agent::Instrumentation::GRPC::Helper
+  end
+
+  def unwanted_host_patterns
+    [/unwanted/.freeze].freeze
+  end
+
+  def test_class
```

Comment:
Just wondering—I know MiniTest looks at methods starting with `test_`, and even though this doesn't look like its testing anything, its still cool to name like this? I also could be misunderstanding this.

Reply:
That's a great point, @hannahramadan - I don't want MiniTest to see that helper method as a test. With [3f53ed2](https://github.com/newrelic/newrelic-ruby-agent/pull/1283/commits/3f53ed2a2a820052276411d6d60c679bc4845f24) I've renamed the method to `helped_class`.